src/library/basesqltablemodel.cpp

```diff
@@ -20,7 +20,9 @@
 #include "util/dnd.h"
 #include "util/assert.h"
 
-const bool sDebug = false;
+static const bool sDebug = false;
+static const int kIdColumn = 0; // Must not be changed
```

Comment:
The code in master supports the id column being in any spot in the table column list though.